spec/adapters/publishing_adapter_spec.rb

```diff
@@ -560,7 +560,7 @@
       manual.sections = [section]
       manual.removed_sections = [removed_section]
 
-      allow(removed_section).to receive(:withdrawn?).and_return(false)
+      allow(removed_section).to receive(:published?).and_return(true)
```

Comment:
It looks as if the `if !section.withdrawn?` condition still exists in the implementation. How come you've *replaced* the stubbing of `Section#withdrawn?` with the stubbing of `Section#published?` rather than stubbing combinations of both (introducing new examples where necessary)...?